index.html

```diff
@@ -222,30 +257,35 @@ <h3>Skills</h3>
                     <li>Gitlab</li>
                     <li>Terminal</li>
                 </ul>
+            </div>
             </ul>
             </div>
         </section>
-        <section class="contact">
-            <div>
+        <section class="contact" id="Contact">
+            <div class="contact-content">
             <h2>I'm always interested in hearing about new projects, 
                 so if you'd like to chat please get in touch.
             </h2>
             <form action="https://formspree.io/f/xeqdywdz" method="post">
-                <input required type="text" placeholder="Full name" maxlength="30">
-                <input required type="email" placeholder="Email address" name="email">
+                <div class="field">
+                    <input required type="text" placeholder="First name" maxlength="30">
+                    <input required type="text" placeholder="Last name" maxlength="30">
+                </div>
+                <input required type="text" placeholder="Full name" maxlength="30" class="hide">
+                <input required type="email" placeholder="Email address" name="email" class="email">
                 <textarea required class="inputfield" type="text" placeholder="Enter text here" maxlength="500"></textarea>
                 <button type="submit">Get in touch</button>
```

Comment:
- [ ] I think it would be better to add the button interaction styling to page for the best user experience :+1: and also to replicate the Figma design

you may kindly visit this[ link ](https://www.geeksforgeeks.org/css-buttons/)your more info on CSS buttons :+1: 

![image](https://user-images.githubusercontent.com/102757126/208063369-471d929c-cbf4-443e-8447-b6e515017bc2.png)

Reply:
- [ ] Great implementing the changes but the styling for the `hover`pseudo class is not yet according to the given template. 👍 👍 

```
color:  #F55800;
background: #FFF5E1;
border: 1px solid #F55800;
```